.idea/codeStyles/Project.xml

```diff
@@ -1,206 +1,8 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
```

Comment:
Please make sure that when you are sending a pr that the .idea files are not committed. Only the files you made changes to should be committed and pushed.

Reply:
Yes I am changing it. Give me half an hour.

Reply:
Read my comment above first. Before you continue working on this, ask @yash1ts as the issue is assigned to him. If he is working on it, there is no point in you sending a pr.

Reply:
@yashk2000 , I have commented on the original issue.